src/ml_tooling/baseclass.py

```diff
@@ -208,40 +250,12 @@ def make_prediction(
             )
             raise MLToolingError(message) from None
 
-    @property
-    def default_metric(self):
-        """
-        Defines default metric based on whether or not the estimator is a regressor or classifier.
-        Then :attr:`~ml_tooling.config.DefaultConfig.CLASSIFIER_METRIC` or
-        :attr:`~ml_tooling.config.DefaultConfig.CLASSIFIER_METRIC` is returned.
-        If passed estimator is a Pipeline, assume last step is the estimator.
-
-        Returns
-        -------
-        str
-            Name of the metric
-
-        """
-
-        return (
-            self.config.CLASSIFIER_METRIC
-            if self.is_classifier
-            else self.config.REGRESSION_METRIC
-        )
-
-    @default_metric.setter
-    def default_metric(self, metric):
-        if self.is_classifier:
-            self.config.CLASSIFIER_METRIC = metric
-        else:
-            self.config.REGRESSION_METRIC = metric
-
     @classmethod
     def test_estimators(
         cls,
         data: Dataset,
         estimators: Sequence,
-        metric: Optional[str] = None,
+        metrics: Union[str, List[str]] = "default",
```

Comment:
Does this limit us to only using named sklearn metrics?

Reply:
At the moment, yes. This was also the behaviour previously, but we can create an issue to modify this in the future. Would probably be handled by a `.from_callable` method on metric